db/seeds.rb

```diff
@@ -65,3 +65,713 @@
 #
 # job = Job.create(job_params)
 # job.post_online!
+
+
+# add skills
+Tag.create(name: "A# .NET", is_approved: true)
```

Comment:
Super expensive seeds.rb document.
Try instead to have all values in an array, then loop through the array and create all of the values in one go.